scripts/run-ranking-evaluation.js

```diff
@@ -0,0 +1,212 @@
+/**
+* This script runs a series of rank-eval calls against the configured index to
+* produce a bunch of numbers from 0 to 1 that represent relevancy performance
+# across a number of metics. We collect these numbers in a spreadsheet to track
+* relevancy performance over time.
+*
+* Usage:
+*   nvm use; ENV=qa node scripts/run-ranking-evaluation.js; pbcopy < out.csv
+*
+*   .. Then, paste the row into the next position in this spreadsheet:
+*   https://docs.google.com/spreadsheets/d/12uDDxi3bInneZwmUQwEiYYkHheNkqW0WT5jxWvix4BE/edit?pli=1&gid=1424571383#gid=1424571383
+*
+*   After pasting, copy col C from the previous row into the newly pasted row
+*   (because the forumla uses relative references).
+*
+*/
+const fs = require('fs')
+const YAML = require('yaml')
+const { fromIni } = require('@aws-sdk/credential-providers')
+const { stringify } = require('csv-stringify/sync')
+
+const { setCredentials: setKmsCredentials } = require('../lib/kms-helper')
+const { loadConfig } = require('../lib/load-config')
+
+// If you need to run this on a ES 5.3 index, you'll need to use branch
+// `pre-es-work-snapshot` and set the following to false because the ES client
+// needs to be v7:
+const v8Client = true
+let esClient
+if (v8Client) {
+  esClient = require('../lib/elasticsearch/client').esClient
+} else {
+  esClient = require('../lib/es-client').esClient
+}
+
+// Pass in object so we get a reference to private methods like `buildElasticQuery`
+const resourcesPriv = {}
+require('../lib/resources')({}, resourcesPriv)
```

Comment:
Should this method be public?

Reply:
The whole way that this module attaches itself to the app and exposes public/private methods ought to be changed, but not as part of this work.